modules/comparing-runs.adoc

```diff
@@ -0,0 +1,51 @@
+:_module-type: PROCEDURE
+
+[id='comparing-runs_{context}']
+= Comparing runs
+
+[role='_abstract']
+You can compare up to 10 pipeline runs at one time, and view available parameter, scalar metric, confusion matrix, and receiver operating characteristic (ROC) curve data for all selected runs.
+
+.Prerequisites
+* You have logged in to {productname-long}.
+ifdef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+ifndef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group} ) in OpenShift.
+endif::[]
+* You have previously created a data science project that is available and has a pipeline server.
+* You have imported a pipeline to an active pipeline server.
+* You have created at least 2 pipeline runs.
+
+.Procedure
+. In the {productname-short} dashboard, select *Experiments > Experiments and runs*.
++ 
+The *Experiments* page opens.
+. From the *Project* drop-down list, select your data science project.
```

Comment:
What the user is actually selecting here is the project that contains the relevant runs to be compared.

Reply:
Updated.